src/main.ts

```diff
@@ -445,10 +445,19 @@ function update_telem_health(health_packet: any) {
 
 }
 
-ipcRenderer.on("common", (evt, msg) => {
+function update_heartbeat(hb_packet: any) {
+    document.getElementById("drone_voltage").innerText = hb_packet['battery_voltage'] + " V"
+    document.getElementById("drone_rssi").innerText = hb_packet['rssi'] + " dBm"
+}
+
+ipcRenderer.on("common_gss", (evt, msg) => {
     update_telem_health(msg)
 })
 
+ipcRenderer.on("common_hb", (evt, msg) => {
+    update_heartbeat(msg['data'])
+})
+
```

Comment:
What is this additional empty line? Can you even code? You stupid idiot shouldn't even have touched a computer, I can't believe anyone will allow you contribute code here!!!!11!!!

Reply:
that's it. 
![image](https://github.com/ISSUIUC/GroundStation/assets/143204753/b9612282-c363-48ff-b88b-a3db24d22386)